Source/ZoomNet/Models/PhoneDetailNumberType.cs

```diff
@@ -0,0 +1,27 @@
+using System.Text.Json.Serialization;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Specifies the type of phone number in the Zoom Phone API.
+	/// </summary>
+	/// <remarks>
+	/// <list type="table">
+	/// <item><term>toll</term><description>Toll number.</description></item>
+	/// <item><term>tollfree</term><description>Toll-free number.</description></item>
+	/// </list>
+	/// </remarks>
+	[JsonConverter(typeof(JsonStringEnumConverter))]
```

Comment:
Not necessary. The `EnumConverterFactory` is already registered to serialize/deserialize all `enum` using their corresponding `StringEnumConverter<T>`